### DISCUSSION TITLE
Why is Zen Twilight's floating URL bar text much smaller than Firefox's?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [zen-browser/desktop#3459] [General]
<img width="1496" alt="CleanShot 2024-12-10 at 02 53 11@2x" src="https://github.com/user-attachments/assets/c84087e9-5648-43d7-9bda-5cdbbb38e56b">

**Comment by vraravam** [comment]
you are comparing the url bar within the sidebar in Zen vs the urlbar in the conventional/traditional location on top in FF. If you switch to the traditional location in Zen, the floating url bar in Zen is bigger in width.

**Reply by MoltenMonster** [reply]
No that's a floating URL bar, that's not inside the sidebar. It shows in Zen Twilight no matter what the window size is. These are both of the menus that show when you press `CTRL L` or press the URL bars.

**Reply by vraravam** [reply]
you can go to `about:config` and change `zen.view.use-single-toolbar` to `false`. It will then show up like this:
<img width="1760" alt="Screenshot 2024-12-10 at 15 25 15" src="https://github.com/user-attachments/assets/4b0508cd-cb55-42d1-b0bf-7b312f3c76f2">

**Reply by MoltenMonster** [reply]
Ew no, I want everything on the side since it looks better & gives more horizontal space. I'm just asking why the text in the floating menu is intentionally made smaller than on Firefox, it makes it a bit harder to read.

**Comment by tonilampela** [comment]
I might ride on this thread, as I think many places in browser chrome, icons and text are super tiny. All text and icons in side bar for example are very illegible, at least on MacBook Pro 14" with built in screen and with Apple Studio Display.

Which is strange because for example in URL field the font is normal size (compared to Github tab's text):

![CleanShot 2024-12-23 at 15 23 30@2x](https://github.com/user-attachments/assets/9c560a3d-9dee-4ee5-8ecf-00f5ab717db2)
